src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery randomBall = new Lottery();
```

Comment:
don't  name "Lottery" as "Ball" it's super confusing given you have Ball model

Reply:
i can used class Lottery but with other name or need using class Ball?